Resources/config/exporter.xml

```diff
@@ -1,6 +1,30 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
-        <service id="sonata.core.exporter" class="Sonata\CoreBundle\Exporter\Exporter"/>
+        <service id="sonata.core.exporter.writer.csv" class="Exporter\Writer\CsvWriter" public="false">
+            <argument>php://output</argument>
+            <argument>,</argument>
```

Comment:
Hmm does not `CsvWriter` have default values for that? Maybe same for the first argument.

BTW, you made the services private. But what if the user want to change this?